content/chat/getting-started/javascript.textile

```diff
@@ -0,0 +1,198 @@
+
+async function getStarted() {
+
+  const realtimeClient = new Ably.Realtime({ key: '{{API_KEY}}', clientId: 'my-first-client' });
+
+  const chatClient = new ChatClient(realtimeClient);
+
+  const { off } = chatClient.connection.onStatusChange((change) => console.log(`Currently ${change.current}!`));
+
+};
+
+getStarted();
+```
+
+You can monitor the lifecycle of clients' connections, but for now just log a message to the console to know that the connection attempt was successful. You'll see a message saying @Currently connected!@ printed to your console.
+
+
+h2(#step-2). Step 2: Create a room and send a message
+
+Messages are how your clients interact with one another. Use rooms to separate and organize clients and messages into different topics, or 'chat rooms'. Rooms are the entry object into Chat, providing access to all of its features, such as messages, presence and reactions.
+
+* Add the following lines to your @getStarted()@ function to create an instance of a room, attach to the room instance, and then register a listener to subscribe to messages sent to the room. You then also send your first message. Afterwards, run it:
+
+```[javascript]
+const room = await chatClient.rooms.get('my-first-room', AllFeaturesEnabled);
+
+await room.attach();
+
+const { unsubscribe: messageUnsubscribe } = room.messages.subscribe((message: MessageEvent) => {
+  console.log(`Received message: ${ message.message.text }`);
+});
+
+const myFirstMessage = await room.messages.send({ text: 'My first message!' });
+```
+
+* Use the Ably CLI to send a message to your first room: @ably rooms messages send my-first-room 'Hello!'@. The message will be received by the client you've subscribed to the room, and be logged to the console.
+
+h2(#step-3). Step 3: Edit a message
+
+If your client makes a typo, or needs to update their original message then they can edit it.
+
+* Add the following line to your @getStarted()@ function to edit your original first message. Technically this will be your second message that you edit, as running the function again will re-send the original message.
+
+```[javascript]
+await room.messages.update(myFirstMessage.copy({ text: 'My 1st message! (edited)' }));
+```
+
+
+h2(#step-4). Step 4: Show who is typing a message
+
+Typing indicators enable you to display messages to clients when someone is currently typing. An event is emitted when someone starts typing, when they press a keystroke, and then another event is emitted after a configurable amount of time has passed without a key press.
+
+* In practice the @typing.start()@ method would be called on keypress, however for demonstration purposes we will call it and wait for the default period of time to pass for a stop event to be emitted. Add the following lines to @getStarted@ to subscribe to typing events and emit one, then run it:
+
+```[javascript]
+const { unsubscribe: typingUnsubscribe } = room.typing.subscribe((typingEvent) => {
```

Comment:
Same as before about `unsubscribe`.

Can the event just be called `event` instead of `typingEvent`? Seems obvious from context what event it is.